cert/aziot-certd/src/lib.rs

```diff
@@ -784,3 +820,12 @@ fn get_cert_inner(
     let bytes = load_inner(&path)?;
     Ok(bytes)
 }
+
+fn principal_to_map(
+    principal: Vec<Principal>,
+) -> std::collections::BTreeMap<libc::uid_t, Vec<String>> {
```

Comment:
Make this a `BTreeMap<libc::uid_t, Vec<wildmatch::WildMatch>` so that you don't have to parse the `String` into a `WildMatch` again and again for every request.

Reply:
Fixed